build.sbt

```diff
@@ -55,8 +55,8 @@ lazy val cassandra =
     .settings(Defaults.itSettings)
     .settings(Dependencies.cassandra)
     .dependsOn(core)
-    .dependsOn(coreTest % "test->test;compile->test;")
-    .dependsOn(testkit % "compile->compile;test->test")
+    .dependsOn(coreTest % "test->compile;it->test")
+    .dependsOn(testkit % "test->compile;it->compile")
```

Comment:
Shouldn't it be `test->test`? 

As I understand it, it is saying that the cassandra module depends on the test code in `coreTest` and `testKit`, but it should depend only on test, no? Or am I flipping things around?

Reply:
Or maybe it should be `compile->test` because we are want the code from coreTest and testkit available in test scope

Reply:
I think I tried various combination, but maybe I got it wrong. The pom.xml is correct with this anyway.

Reply:
Ok, let's go for it. 

I restart the build. Failed because of some flakiness.